### PR TITLE
New draft of Working Group RFC

### PR DESCRIPTION
**Pull request by XAMPPRocky**
### [Rendered](https://github.com/rust-lang/wg-governance/blob/project-wg-rfc-next/draft-rfcs/working-group-terminology.md)

cc @rust-lang/wg-governance

### CODE REVIEW

**Review by BatmanAoD — APPROVED** [review]

**Comment by nikomatsakis** [comment]
Did we plan to include the "workflow" that we talked about last meeting?

**Comment by XAMPPRocky** [comment]
@nikomatsakis This is the draft from before that meeting. I've yet to include it. I'm planning on working on it tomorrow.

File: draft-rfcs/working-group-terminology.md [hunk 1]
```diff
@@ -54,6 +65,78 @@ Group" term, into three distinct terms.
...
+  handling permissions.
+
+4. Create a post on the Rust or Inside Rust blog announcing creation of
+   the group.
+
+5. The group works towards the goals laid out in their charter.
+
+6. When active work has stopped a group is "archived".
+
+- Archival is a not permanent state, it is only a reflection on the current
```
Comment (nikomatsakis) [inline draft-rfcs/working-group-terminology.md]:
```suggestion
- Archival is not necessarily a permanent state, it is only a reflection on the current
```

**Review by nikomatsakis — APPROVED** [review]
Looks great!

**Comment by valgrimm** [comment]
Propose this flow chart be added on line 74.
FLOW-rustprojectgroups-v1

**Comment by valgrimm** [comment]
Revised flow chart
FLOW-rustprojectgroups-v2

**Comment by XAMPPRocky** [comment]
Merging as this is now https://github.com/rust-lang/rfcs/pull/2856

**Pull request merged**